docs/reference/indices/get-data-stream.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[indices-get-data-stream]]
+=== Get data stream API
+++++
+<titleabbrev>Get data stream</titleabbrev>
+++++
+
+Returns information about one or more data streams.
+
+////
+[source,console]
+-----------------------------------
+PUT /_data_stream/my-data-stream
+{
+  "timestamp_field" : "@timestamp"
+}
+-----------------------------------
+// TESTSETUP
+////
+
+////
+[source,console]
+-----------------------------------
+DELETE /_data_stream/foo
+-----------------------------------
+// TEARDOWN
+////
+
+[source,console]
+--------------------------------------------------
+GET _data_streams/my-data-stream
+--------------------------------------------------
+
+[[get-data-stream-api-request]]
+==== {api-request-title}
+
+`GET _data_streams/<data-stream>`
```

Comment:
I think this should be singular:

```suggestion
`GET _data_stream/<data-stream>`
```

Reply:
It is `_data_streams` (plural) which I see is inconsistent with many of the other endpoints such as `_alias`, `_index_template`, `_security/role`, etc., that use the singular for retrieving information about one or more resources. @martijnvg, do you think we should change this?

Reply:
Yes, I think we should change this to `_data_stream` in order to make this consistent with other apis.

Reply:
Fyi, we will change this endpoint to use the singular `_data_stream` in a series of follow-up PRs.